helix-core/src/main/java/org/apache/helix/monitoring/mbeans/JobMonitor.java

```diff
@@ -41,6 +45,12 @@
   private static final Logger LOG = LoggerFactory.getLogger(JobMonitor.class);
   private static final long DEFAULT_RESET_INTERVAL_MS = 60 * 60 * 1000; // 1 hour
 
+  // The root path of ZNodes that are used to sync certain metrics to ZooKeeper. This path is the
+  // first level, and its children nodes will represent each job type. Since the metrics are
+  // aggregated by job types, the metric values will be recorded in each ZNode belonging to a job
+  // type as simple fields.
+  private static final String SYNC_ZK_ROOT_PATH = "/JOB_MONITOR_METRICS";
```

Comment:
Let's review whether creating this root path would be appropriate. Can you find a better place to persist the data?